open_schools_platform/organization_management/circles/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+from typing import Any
+
+import safedelete
+from django.contrib.gis.geos import Point
+from django.core.validators import MinValueValidator
+from django.contrib.gis.db import models
+from simple_history.models import HistoricalRecords
+
+from open_schools_platform.common.models import BaseModel, BaseManager
+from open_schools_platform.organization_management.organizations.models import Organization
+
+
+class CircleManager(BaseManager):
+    def create_circle(self, *args: Any, **kwargs: Any):
+        circle = self.model(
+            *args,
+            **kwargs,
+        )
+        circle.full_clean()
+        circle.save(using=self._db)
+
+        return circle
+
+
+class Circle(BaseModel):
+    _safedelete_policy = safedelete.config.SOFT_DELETE_CASCADE
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    name = models.CharField(max_length=200)
+    organization = models.ForeignKey(Organization, related_name="circles", on_delete=models.CASCADE)
+    objects = CircleManager()
+    capacity = models.IntegerField(validators=[MinValueValidator(0)], default=0)
+    address = models.CharField(max_length=255, default="")
+    description = models.CharField(max_length=2000, default="")
+    location = models.PointField(geography=True, default=Point(0.0, 0.0))
```

Comment:
Yes, this is strange. I think django demanded default value for this field and I didn't want to set null=True because in fact this field is not nullable. So I decided to set this value as default but this solution isn't great. 

What do u think, which option is better?

Reply:
I don't know. We should think about that.